### ISSUE TITLE
Dependancies

### ISSUE THREAD

**Issue by MaxPowerz** [ArcadeMachinist/StarwingParadox#2]
Just curious as to what you use to start this server and its dependancies.
I tried using Node,js, but it throws an error saying that it can't find node-fetch

**Comment by catchra** [comment]
@MaxPowerz 
I only just found this myself so I haven't had a chance to work with it but from the looks of it it's not a node.js application but instead a web service from what I've seen after looking through the files so far you would need an nginx web server and you would need a PostgreSQL 12 database.

but on the side as part of my own observation there would be no way to directly tell the game to use this as the game by default wants to connect to dev.starwing.jp so you would need a way to redirect those requests to either a server you control or to LocalHost

**Comment by MaxPowerz** [comment]
@catchra 
Yes, it needs Nginx, posgress, and it also needs some work to fix a few issues, Like the tutorial for 8v8 never gets flagged as completed so it wont unlock all the maps in 8v8.

**Comment by ArcadeMachinist** [comment]
More than that - pure StarWing server is just a part of the whole setup.
It also needs NESICA server with correct game admission settings.
You may google "Nesica at Home", but that requires substencial amount of work.
However my design is based on heavily modified version of that thing.
For copyright reasons I can not share Nesica part of my setup.
But you can run "FakeNesicaService" instead, see my other repos.

Again, it requires some legwork and understanding on how Taito games are using that service layer.
